### PR TITLE
Updated Schema

### PR DESCRIPTION
**Pull request by krausf**
This PR resembles the Entities and Properties for MVP as documented in the GoogleDoc.

Ontologies and assignment can still be object to change.

This PR includes:

- addition of slots and descriptions
- Definition of the Sequencing and LibraryPreparation Protocol

### CODE REVIEW

File: src/schema/ghga.yaml [hunk 1]
```diff
@@ -196,8 +209,17 @@ classes:
       type:
         description: >-
           The type of Study. For example, "Cancer Genomics", "Epigenetics", "Exome Sequencing".
-        # TODO: Enum
+        range: study_type_enum
+        required: true
+      short name:
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:214]: I am not sure if short name is really helpful. For whom is the short name? The data submitter? data approver? End user?

File: src/schema/ghga.yaml [hunk 2]
```diff
@@ -278,15 +305,20 @@ classes:
       used and the Agent that is running the experiment.
     slots:
       - title
+      - name
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:308]: I think it is misleading to have both title and name. We should have either one or the other.

File: src/schema/ghga.yaml [hunk 3]
```diff
@@ -402,6 +504,7 @@ classes:
     slots:
       - name
       - description
+      - external sample id
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:507]: What are the values that this field will hold? Like EBI BioSamples IDs?

**Review by deepakunni3 — CHANGES_REQUESTED** [review]
Looks great! Few small changes noted.

File: src/schema/ghga.yaml [hunk 4]
```diff
@@ -154,11 +154,19 @@ classes:
     close_mappings:
       - NCIT:C47885
     slots:
+      - description
+      - has publication
```
[the remark this answers is not in the document]
Comment (krausf) [reply src/schema/ghga.yaml:158]: Solved

File: src/schema/ghga.yaml (hunk 2 above)
Comment (krausf) [reply src/schema/ghga.yaml:308]: Solved

**Review by deepakunni3 — APPROVED** [review]
Thanks for the quick turn around @krausf ! 🙌

**Pull request merged**